### ISSUE TITLE
Setup CGAL

### ISSUE THREAD

**Issue by Yinchu-Liu** [Simple-XX/SimplePhysicsEngine#1]
Does there exist an easier way to install CGAL (Probably without using CMake) ? I tried to use CMake to install CGAL but it failed and gave me this error:(

```
CMake Error at CMakeLists.txt:7 (find_package):
  By not providing "FindCGAL.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "CGAL", but
  CMake did not find one.

  Could not find a package configuration file provided by "CGAL" with any of
  the following names:

    CGALConfig.cmake
    cgal-config.cmake

  Add the installation prefix of "CGAL" to CMAKE_PREFIX_PATH or set
  "CGAL_DIR" to a directory containing one of the above files.  If "CGAL"
  provides a separate development package or SDK, be sure it has been
  installed.
```

**Comment by GregoryLi0** [comment]
In cmake-gui, set 'CMAKE_MODULE_PATH' to dir of CGAL, for example 'D:\xxxx\dev\CGAL-5.4'.

**Comment by MRNIU** [comment]
fixed #18
